crates/configuration/src/global_settings.rs

```diff
@@ -1,4 +1,8 @@
-use crate::shared::types::{Duration, IpAddress, MultiAddress};
+use std::{net::IpAddr, str::FromStr};
+
+use multiaddr::Multiaddr;
```

Comment:
I think you should keep the naming as it was if possible - there is no reason for shortening them

```suggestion
use std::{net::IpAddress, str::FromStr};

use multiaddr::Multiaddress;
```